python/ray/_raylet.pyx

```diff
@@ -1167,6 +1167,26 @@ cdef class CoreWorker:
             check_status(CCoreWorkerProcess.GetCoreWorker().Delete(
                 free_ids, local_only))
 
+    def get_object_locations(self, object_refs, int64_t timeout_ms):
```

Comment:
Does it make sense to call it `get_object_information`? And I think in the highest level layer, we can make two methods (`get_object_size` and `get_object_location` which parses different data from this internal method).

Reply:
seems the convention is to have object_size info in the `Location`. happy to change if we think it's necessary!